src/components/Card/index.js

```diff
@@ -12,6 +12,10 @@ const Card = (props) => {
     tempCards: {},
   });
 
+  useEffect(() => {
+    console.log(props.card.isChecked);
+  }, []);
+
```

Comment:
Ideally you should not have any warnings in the terminal/console log:
![image](https://user-images.githubusercontent.com/52723192/102369714-51e8f800-3fcd-11eb-8eed-fac11512b665.png)